web/src/lib/components/shared-components/upload-panel.svelte

```diff
@@ -29,9 +34,16 @@
 		out:fade={{ duration: 250, delay: 1000 }}
 		on:outroend={() => {
 			notificationController.show({
-				message: 'Upload success, refresh the page to see new upload assets',
+				message: `Upload success, refresh the page to see new upload assets`,
 				type: NotificationType.Info
 			});
+			if (isUploadingWithDuplicate) {
+				notificationController.show({
+					message: `Duplicate pictures will not appear on refresh`,
+					type: NotificationType.Warning
+				});
```

Comment:
I don't like this message, personally. I don't think it is very clear of informative. What do you think about changing `isUploadingHasDuplicate` to `duplicateCount`? It gets incremented for each duplicate, instead of setting the value to `true`. When the upload finishes, if the duplicate count is bigger than zero, we show a warning message "Skipped {duplicateCount} duplicate pictures".